docs/en/integrations/data-ingestion/kafka/cloud/confluent/index.md

```diff
@@ -135,7 +137,7 @@ The instructions for creating an HTTP Sink in Confluent Cloud can be found [here
 * HTTP server details - The connection details for ClickHouse. Specifically:
     * `HTTP Url` - This should be of the same format as the self-managed configuration parameter `http.api.url i.e. &lt;protocol>://&lt;clickhouse_host>:&lt;clickhouse_port>?query=INSERT%20INTO%20&lt;database>.&lt;table>%20FORMAT%20JSONEachRow`
     * `HTTP Request Method` - Set to POST
-    * `HTTP Headers` - “Content Type: application/json”
+    * `HTTP Headers` - `Content Type: application/json`
```

Comment:
Our research yesterday showed that it need to be removed

Reply:
I tested with the header, and everything works. But in general, we can remove it since it's not mandatory header for the server

Reply:
Les work for the user is better